### PR TITLE
NTAG213TT Tamper Feature Support

### PR DESCRIPTION
**Pull request by mjaksn**
For NTAG213TT, includes tamper feature related info in `hf mfu info` and adds `hf mfu tamper` commands for configuring the tamper feature

### CODE REVIEW

**Comment by mjaksn** [comment]
Shoot, let me get that fixed and I'll add a new commit

**Comment by iceman1001** [comment]
```
  905 |     uint8_t tt_msg_resp_len;
   should be initialized
   
     905 |     uint8_t tt_msg_resp_len = 0;
```

**Comment by mjaksn** [comment]
Thanks! I think that should address both

File: client/src/cmdhfmfu.c [hunk 1]
```diff
@@ -2529,6 +2679,141 @@ static void wait4response(uint8_t b) {
...
+    int tt_msg_page = 45;
+    int msg_len = 0;
+    uint8_t msg_data[4] = {0x00};
+    CLIGetHexWithReturn(ctx, 3, msg_data, &msg_len);
+    bool use_msg = (msg_len > 0);
+
+    if (use_msg && msg_len != 4) {
+        PrintAndLogEx(WARNING, "The tamper message must be 4 hex bytes if provided");
+        DropField();
+        return PM3_ESOFT;
```
Comment (iceman1001) [inline client/src/cmdhfmfu.c:2717]: CLIParserFree(ctx);

**Comment by iceman1001** [comment]
Almost...   
I put a comment where its needed too

**Comment by mjaksn** [comment]
appreciate the help with those memory issues

**Comment by iceman1001** [comment]
Awesome contribution!

**Pull request merged**